databases/sqlcipher/Portfile

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           clang_dependency 1.0
+
+name                sqlcipher
+# don't forget to update the checksums for sqlite3-tools when updating sqlite3
+version             4.4.2
+revision            0
+categories          databases
+platforms           darwin
+license             public-domain
+
+maintainers         { gmail.com:ybzhao1989 @z7z8th } openmaintainer
+
+description         an security extesion to the embedded SQL database engine
+
+long_description    SQLCipher extends the SQLite database library to add \
+                    security enhancements that make it more suitable for \
+                    encrypted local data storage like: \
+                     \* on-the-fly encryption\
+                     \* tamper detection\
+                     \* memory sanitization\
+                     \* strong key derivation\
+                    \
+                    SQLite3 is an SQL database engine in a C library. \
+                    Programs that link the SQLite3 library can have SQL \
+                    database access without running a separate RDBMS \
+                    process. The distribution comes with a standalone \
+                    command-line access program (sqlite3) that can be used \
+                    to administer an SQLite3 database and which serves as \
+                    an example of how to use the SQLite3 library.
+
+homepage            https://www.zetetic.net/sqlcipher/
+master_sites        https://github.com/sqlcipher/sqlcipher/archive/refs/tags/
+
+distname            v${version}
+
+checksums           rmd160  bc0a00765058b864a95686d9efe9f24a2e68f6cb \
+                    sha256  87458e0e16594b3ba6c7a1f046bc1ba783d002d35e0e7b61bb6b7bb862f362a7 \
+                    size    17611236
+
+depends_lib         port:libedit \
+                    port:ncurses \
+                    port:zlib
+
+if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"} {
+    # This port is in the dependency chain for clang 3.7 and later
+    clang_dependency.extra_versions 3.7
+}
+
+configure.args      --enable-threadsafe \
+                    --enable-dynamic-extensions \
+                    --disable-readline \
+                    --enable-editline \
+                    --enable-tempstore=yes \
+                    AWK=/usr/bin/awk
+
+configure.cppflags-append  -DSQLITE_HAS_CODEC
+
+build.type          gnu
+
+# livecheck.type      regex
+# livecheck.url       https://www.zetetic.net/sqlcipher/verify/
+# livecheck.regex     sqlcipher-(4(?:\\.\[0-9\]+)+).zip
```

Comment:
please don't add commented out lines; switching to the GitHub PG will set this automatically anyway so it can be removed